pkg/cmd/server/api/types.go

```diff
@@ -980,3 +983,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
```

Comment:
That's fine

On Mon, Feb 8, 2016 at 3:03 PM, Luke Meyer <notifications@github.com> wrote:

> In pkg/cmd/server/api/types.go
> <https://github.com/openshift/origin/pull/6901#discussion_r52203403>:
>
> > @@ -980,3 +983,19 @@ type AdmissionConfig struct {
> >  	// on the master. Order is significant. If empty, a default list of plugins is used.
> >  	PluginOrderOverride []string
> >  }
> > +
> > +// PodLimitRequestConfig is the configuration for the PodLimitRequest
> > +// admission controller which overrides user-provided container request/limit values.
> > +type PodLimitRequestConfig struct {
>
> I have no idea what future resource policy might look like, other than
> knowing that soon we'll be setting request/limit on the pod directly rather
> than containers. I could maybe see the scope of this plugin being expanded
> if other resources come into play? Not sure.
>
> I went with ClusterResourceOverride and ClusterResourceOverrideConfig.
> Lemme know if you have any strong feelings about that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6901/files#r52203403>.
>